manager/src/main/java/com/redhat/service/smartevents/manager/api/models/requests/BridgeRequest.java

```diff
@@ -1,16 +1,24 @@
 package com.redhat.service.smartevents.manager.api.models.requests;
 
+import javax.validation.Valid;
 import javax.validation.constraints.NotEmpty;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.redhat.service.smartevents.infra.models.gateways.Action;
+import com.redhat.service.smartevents.manager.api.user.validators.processors.ValidErrorHandler;
 import com.redhat.service.smartevents.manager.models.Bridge;
 
+@ValidErrorHandler
 public class BridgeRequest {
 
     @NotEmpty(message = "Bridge name cannot be null or empty")
     @JsonProperty("name")
     private String name;
 
+    @JsonProperty("errorHandler")
+    @Valid
+    private Action errorHandler;
```

Comment:
A nitpick, AFAIK the `errorHandler` should be also in BridgeResponse, so users can see error handler in Bridge.

Reply:
good catch @sutaakar . done now :+1: